modules/Rubrics/src/Visualise.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/*
+Gibbon, Flexible & Open School System
+Copyright (C) 2010, Ross Parker
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Gibbon\Module\Rubrics;
+
+use Gibbon\Contracts\Database\Connection;
+use Gibbon\session;
+use Gibbon\UI\Chart\Chart;
+
+/**
+ * Attendance display & edit class
+ *
+ * @version 12th Sept 2016
+ * @since   12th Sept 2016
+ * @author  Sandra Kuipers
```

Comment:
Looks like a copy-pasted file 😂 I didn't think my name was still lurking on any of these file headers, gotta fix that.

Reply:
Hahaha, not as thorough as I thought I was being. Fixed now.